package.json

```diff
@@ -28,6 +32,8 @@
 		"test": "vitest run",
 		"dev": "vitest",
 		"local-release": "changeset version && changeset publish",
-		"prepublishOnly": "npm run ci"
+		"prepublishOnly": "npm run ci",
+		"commitlint": "npx commitlint --edit",
+    "lint-staged": "npx lint-staged"
```

Comment:
There is no need for `lint-staged` since you can run your lefthook job on staged files also, you check this [ISSUE](https://github.com/evilmartians/lefthook/issues/140) as reference ;)